powerquery-docs/Connectors/Funnel.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: Power Query Funnel connector
-description: Provides basic information, prerequisites, and information on how to connect to your Funnel data, along with a list of known issues and limitations.
-author: bezhan-msft
+description: Provides basic information, prerequisites, and information on how to connect to your data, along with a list of known issues and limitations.
```

Comment:
Why was "Funnel" removed here?

Reply:
Funnel was removed seeing as it is not Funnel data. The data is in fact the customers data from 3rd parties.